templates/layout.html

```diff
@@ -15,7 +15,7 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <meta name="description" content="{{description}}">
     <link rel="search" type="application/opensearchdescription+xml" href="/opensearch.xml" title="cdnjs">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.1.1-1/css/united/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.4/united/bootstrap.min.css">
```

Comment:
3.1.1-1 doesn't include the responsive-embed class. It didn't seem like bumping the version breaks anything, but we could just include the classes in theme.css if we want to add them manually. They aren't much.

```css
.embed-responsive {
    position: relative;
    display: block;
    height: 0;
    padding: 0;
    overflow: hidden
}

.embed-responsive .embed-responsive-item, .embed-responsive embed,
.embed-responsive iframe, .embed-responsive object, .embed-responsive video {
    position: absolute;
    top: 0;
    bottom: 0;
    left: 0;
    width: 100%;
    height: 100%;
    border: 0
}

.embed-responsive-16by9 {
    padding-bottom: 56.25%
}

.embed-responsive-4by3 {
    padding-bottom: 75%
}
```